pkgs/by-name/cr/criticality_score/package.nix

```
@@ -0,0 +1,34 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "criticality_score";
+  version = "2.0.2";
```

```suggestion
  version = "2.0.4";
```
New version released by now